### DISCUSSION TITLE
Android Bundling failed 14417ms

### DISCUSSION THREAD

**Discussion by manhimselff** [community/community#135607] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

After i run the expo start i keep running into this error 
(Android Bundling failed 14417ms C:\Users\KTI - LLC\Desktop\Airlormet Project\AirlormetStoreApp\node_modules\expo\AppEntry.js (868 modules)
The package at "node_modules\cipher-base\index.js" attempted to import the Node standard library module "stream".
It failed because the native React runtime does not include the Node standard library.)

**Comment by Akash1134** [comment]
Hi @manhimselff ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
